### PR TITLE
feat!: Upgrade Dotnet Dictionary

### PR DESCRIPTION
**Pull request by KieranRobson**
# Add/Fix Dictionary

Dictionary: `Dotnet`

## Description

Upgrade as part of #705 

## References

NA

## Checklist

- [x] By submitting this issue, you agree to follow our
      [Code of Conduct](https://github.com/streetsidesoftware/cspell-dicts/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: dictionaries/dotnet/package.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
     "./cspell-ext.json": "./cspell-ext.json"
   },
   "scripts": {
-    "build": "cspell-tools compile dotnet.txt --no-compress -o ./dict",
-    "test": "head -n 100 \"dotnet.txt\" | cspell -v -c ./cspell-ext.json --local=* --languageId=cs stdin",
+    "build": "cspell-tools-cli compile src/dotnet.txt --no-compress -o ./dict",
```
Comment (Jason3S) [inline dictionaries/dotnet/package.json:14]: Please use `--keep-raw-case`

Comment (KieranRobson) [reply dictionaries/dotnet/package.json]: Done!

**Comment by Jason3S** [comment]
@KieranRobson,

You might need to run `yarn build` in that directory.

**Comment by KieranRobson** [comment]
```
user@BOOK-9IDCLB5GQI MINGW64 ~/OneDrive/Projects/Misc Project/cspell-dicts/dictionaries/dotnet (upgrade-dotnet)
$ yarn run build
yarn run v1.22.19
$ cspell-tools-cli compile --keep-raw-case src/dotnet.txt --no-compress -o ./dict
Compile:
 output: ./dict
 compress: false
 files:
  src/dotnet.txt


2022-11-09T14:27:59.392Z Start compile: dotnet
2022-11-09T14:27:59.394Z Reading dotnet.txt
2022-11-09T14:27:59.404Z Done reading dotnet.txt
2022-11-09T14:27:59.408Z Process "src\dotnet.txt" to "dict\dotnet.txt"
2022-11-09T14:27:59.431Z Done processing src\dotnet.txt
2022-11-09T14:27:59.461Z Done "dict\dotnet.txt"
2022-11-09T14:27:59.462Z Done compile: dotnet
2022-11-09T14:27:59.462Z Complete.
Done in 0.39s.

user@BOOK-9IDCLB5GQI MINGW64 ~/OneDrive/Projects/Misc Project/cspell-dicts/dictionaries/dotnet (upgrade-dotnet)
$ yarn run test
yarn run v1.22.19
$ head -n 100 src/dotnet.txt | cspell -v -c ./cspell-ext.json --local=* --languageId=cs stdin

cspell;
Date: Wed, 09 Nov 2022 14:28:06 GMT
Options:
    verbose:   Yes
    config:    ./cspell-ext.json
    exclude:   node_modules/**
    files:     stdin
    wordsOnly: No
    unique:    No

Config Files Found:

Exclusion Globs:
    Glob: node_modules/** from command line

1/1 .\stdinChecking: stdin, File type: cs, Language: *
Checked: stdin, File type: cs, Language: * ... Issues: 0 0.4045086S
Config file Used: C:\Users\user\OneDrive\Projects\Misc Project\cspell-dicts\dictionaries\dotnet\cspell-ext.json
Dictionaries Used: backwards-compatibility, companies, filetypes, public-licenses, softwareTerms, aws, cryptocurrencies, dotnet       
 404.51ms
CSpell: Files checked: 1, Issues found: 0 in 0 files
Done in 0.96s.
```

No changes made when running Build and Test

**Comment by Jason3S** [comment]
@KieranRobson,

I have fixed it in #1617

Thank you for the help.

**Pull request closed without merging**